cinnabar/wrangle.py

```diff
@@ -41,18 +42,18 @@ def read_csv(filepath: pathlib.Path) -> dict:
     return raw_results
 
 
-class RelativeMeasurement(BaseModel):
+class RelativeMeasurement(DefaultModel):
     ligandA: str
     ligandB: str
-    DDG: float
-    uncertainty: float
+    DDG: FloatQuantity['kilojoule_per_mole']
+    uncertainty: FloatQuantity['kilojoule_per_mole']
     computational: bool
 
 
-class AbsoluteMeasurement(BaseModel):
+class AbsoluteMeasurement(DefaultModel):
     ligand: str
-    DG: float
-    uncertainty: float
+    DG: FloatQuantity['kilojoule_per_mole']
```

Comment:
The nice thing about this is currently with the way openff models works is if a users passes in kcal/mol, it will get converted automatically